assets/quick-add.js

```diff
@@ -26,6 +26,11 @@ if (!customElements.get('quick-add-modal')) {
           .then((responseText) => {
             const responseHTML = new DOMParser().parseFromString(responseText, 'text/html');
             this.productElement = responseHTML.querySelector('section[id^="MainProduct-"]');
+            this.productElement.classList.forEach((classApplied) => {
+              console.log(classApplied);
+              if (classApplied.startsWith('color-') || classApplied === 'gradient')
```

Comment:
@andrewetchen I've added support for the gradient to the modal 👍 
As for the border it was already the case in our current version. If you set background-1 to a gradient, then add som borders on medias. You will see the same result: 
![](https://screenshot.click/23-00-8zh9n-85tf6.png)